docs/ampform/LambdaKpi0.ipynb

```diff
@@ -533,13 +533,13 @@
     "        elif symbol.name.startswith(\"m_{N\"):\n",
     "            categorized_sliders_m[\"N*\"].append(slider)\n",
     "\n",
-    "tabs = []\n",
-    "for category in categorized_sliders_m:\n",
-    "    box_m = w.VBox(categorized_sliders_m[category])\n",
-    "    box_gamma = w.VBox(categorized_sliders_gamma[category])\n",
-    "    combined_box = w.HBox([box_m, box_gamma])\n",
-    "    tabs.append(combined_box)\n",
-    "\n",
+    "tabs = [\n",
+    "    w.HBox([\n",
```

Comment:
@shenvitor this is what I meant: the 'hugging' of `([` in the `HBox` constructor. That only works if preview for Ruff is activated for the formatter (not just the linter), see 66a312d.